UpdatedImageProcessing/UpdatedImageProcessing/ShapeDetection/shape_classification.py

```diff
@@ -34,14 +35,14 @@ def load_polar_side_counter(self):
         self.polar_side_minimums = self.polar_side_counter.get_polar_side_minimums()
         self.num_polar_side_minimums = len(self.polar_side_counter.get_polar_side_minimums())
         self.circle_score = self.polar_side_counter.get_circle_score()
+        self.noise_score = self.polar_side_counter.get_noise_score()
         self.origin = self.polar_side_counter.get_origin()
 
     def determine_shape(self):
```

Comment:
This function and all other functions should have documentation in the form of:

```
def determine_shape(self):
    """
    Function description.

    :param x:      Description of variable,
    :type x:       Type of variable,

    :return:       What the function returns.
    """
```

Reply:
For the future, if you have functions like `determine_shape()` that are void (don't return anything) and don't take any inputs, you can just describe them as:

```
def determine_shape(self);
    """
    Function description.

    :return:       nothing.
    """
```

It's just important to describe what the function does for posterity.